modules/api-gateway/main.tf

```diff
@@ -0,0 +1,33 @@
+# main.tf
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "gateway" {
+  source = "./gateway"
+}
+
+module "rotas_api" {
+  source = "./rotas"
+  gateway_id = module.gateway.api_gateway_id
+  resource_id = module.gateway.resource_id
+  rotas = [
+    {
+      path        = "/rota1"
+      http_method = "GET"
+      uri         = "https://exemplo.com/rota1"
+    },
+    {
+      path        = "/rota2"
+      http_method = "POST"
+      uri         = "https://exemplo.com/rota2"
+    },
+    # Adicione quantas rotas você precisar
+  ]
```

Comment:
@DiegoOLiveira-dev quando finalizarmos o Tech Challenge, teremos 3 rotas: 
- `/catalogo` para a nova API de Catálogo de Produtos
- `/pedido` para a nova API de Pedidos
- `/pagamento` para a nova API de Pagamentos

Creio que já podemos deixar 3 rotas pré-criadas.

O endereço do Load Balancer da AWS referente a cada um dos services do k8s infelizmente só saberemos depois que for feito `apply` dos manifestos do K8s. Por hora acho que podemos deixar "https://exemplo.com/" sem problemas e depois substituímos pelo endereço real manualmente lá no Console da AWS, após fazer o `terraform apply`.